pkg/apis/openebs.io/register.go

```diff
@@ -1,5 +1,6 @@
 package openebsio
 
 const (
+	// Point GroupName to the openebs site
```

Comment:
Thanks @andreathniah for this PR !
Can you please change it to something like "Use openebs.io as a GroupName" as openebs.io here is not a site.

Reply:
Woosp @sagarkrsd! I pushed another commit that changed it!